Mathlib/Data/Finsupp/MonomialOrder.lean

```diff
@@ -113,16 +113,6 @@ noncomputable instance {α N : Type*} [LinearOrder α] [OrderedCancelAddCommMono
   le_of_add_le_add_left a b c h := by simpa only [add_le_add_iff_left] using h
   add_le_add_left a b h c := by simpa only [add_le_add_iff_left] using h
 
-theorem Finsupp.lex_lt_iff {α N : Type*} [LinearOrder α] [LinearOrder N] [Zero N]
-    {a b : Lex (α →₀ N)} :
-    a < b ↔ ∃ i, (∀ j, j< i → ofLex a j = ofLex b j) ∧ ofLex a i < ofLex b i :=
-    Finsupp.lex_def
-
-theorem Finsupp.lex_le_iff {α N : Type*} [LinearOrder α] [LinearOrder N] [Zero N]
```

Comment:
If seems `Finsupp.lex_le_iff` has disappeared, could you add add it in `Data/Finsupp/Lex.lean`?